verify/index.ts

```diff
@@ -31,19 +36,38 @@ async function verify() {
     }
 }
 
-// addCerts adds certificates into Notation trust store.
-function addCerts(dir: string) {
-    let trustStores = fs.readdirSync(dir, {withFileTypes: true, recursive: false})
+// configTrustStore configures Notation trust store based on specs.
+// Reference: https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-store
+function configTrustStore(dir: string) {
+    let trustStoreX509 = path.resolve(dir, X509); // .github/truststore/x509
+    if (!fs.existsSync(trustStoreX509)) {
+        throw new Error("cannot find dir: <trust_store>/x509");
+    }
+    let trustStoreTypes = getSubdir(trustStoreX509); // [.github/truststore/x509/ca, .github/truststore/x509/signingAuthority, ...]
+    for (let i = 0; i < trustStoreTypes.length; ++i) {
+        let trustStoreType = path.basename(trustStoreTypes[i]);
+        let trustStores = getSubdir(trustStoreTypes[i]); // [.github/truststore/x509/ca/<my_store1>, .github/truststore/x509/ca/<my_store2>, ...]
+        for (let j = 0; j < trustStores.length; ++j) {
+            let trustStore = trustStores[j]; // .github/truststore/x509/ca/<my_store>
+            let trustStoreName = path.basename(trustStore); // <my_store>
+            let certFile = getFileFromDir(trustStore); // [.github/truststore/x509/ca/<my_store>/<my_cert1>, .github/truststore/x509/ca/<my_store>/<my_cert2>, ...]
+            execSync(`notation cert add -t ${trustStoreType} -s ${trustStoreName} ${certFile.join(' ')}`, { encoding: 'utf-8' });
```

Comment:
This command `execSync` seems dangerous.

Firstly, `certFile` may contain spaces like `hello world.crt`.

Secondly, it is vulnerable to put arbitrary user input to `execSync`. According to https://nodejs.org/api/child_process.html#child_processexecsynccommand-options
> Never pass unsanitized user input to this function. Any input containing shell metacharacters may be used to trigger arbitrary command execution.

Please consider other exec APIs.

Reply:
This comment also applies to other PRs.

Reply:
@shizhMSFT I did a search, looks like for Github Actions, the standard is using [@actions/exec](https://github.com/actions/toolkit/tree/main/packages/exec#actionsexec). Will switch to this module in other PRs as well.